source/features/highest-rated-comment.tsx

```diff
@@ -68,7 +68,7 @@ function linkBestComment(bestComment: HTMLElement): void {
 	}
 
 	const text = select('.comment-body', bestComment)!.textContent!.slice(0, 100);
-	const {hash} = select('a.js-timestamp', bestComment)!;
+	const {hash} = select<HTMLAnchorElement>('a.js-timestamp', bestComment)!;
```

Comment:
?

This should be automatic

Reply:
[`2c6b447` (#4037)](https://github.com/sindresorhus/refined-github/pull/4037/commits/2c6b447dc177d26867d24a1f1d63b4d96ad4493d)

I know it was complaining... Let me try to revert

Reply:
Check the last commit please

Reply:
Can you find which dependency update caused this? It worked until now and select-dom hasn’t been updated (its dependency might have, though)

Reply:
I will do it later added a todo to main post